packages/accordions/src/elements/stepper/Stepper.tsx

```diff
@@ -28,18 +28,20 @@ interface IStaticStepperExport<T, P>
 }
 
 interface IStepperProps extends HTMLAttributes<HTMLOListElement> {
-  /** Used to show the current step and compute completed steps */
+  /** Defines the currently active step */
```

Comment:
I've received questions on whether this index is 0-based or 1-based for the initial step. @jzempel is this something we want to add?

Reply:
Yeah, I think it would be good to provide the base for added clarity. Can you work out the re-wording @Mikaelia?

Reply:
Can do